etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/database/DatabasePartnerMetadataStorage.java

```diff
@@ -163,8 +161,8 @@ PartnerMetadata partnerMetadataFromResultSet(ResultSet resultSet) {
             }
 
             return new PartnerMetadata(
-                    resultSet.getString(METADATA_TABLE_RECEIVED_MESSAGE_ID),
-                    resultSet.getString("sent_message_id"),
+                    resultSet.getString("inbound_report_id"),
```

Comment:
You reverted some but not everything in this file.  This is still `"inbound_report_id"` and should be reverted to `"received_message_id"`.  There are also some send/outbound that haven't been reverted either.

I noticed this [conversation](https://github.com/CDCgov/trusted-intermediary/pull/1601#discussion_r1857181887) from last week.  I think the decision should go the other way with a constant for both.  This mix-up with only half of the columns being reverted would be harder to come by if there was just one constant value to change.

Reply:
Granted, there are other strings in this file that didn't use the original `METADATA_TABLE_RECEIVED_MESSAGE_ID` but could have.  They also should be updated to use a constant because they could benefit from one of the positives of having a private constant in the a Java class: the ability to keep values in sync through a single constant.

Reply:
From my understanding, only the DB related variables or strings should be kept as the old naming convention as that has been kicked to another story regarding database migration. Anything non-DB related has the new naming convention. The reason the constant was removable was for readability purposes. I think it would be better to reintroduce that constant once the DB changes are made, if we do choose to have it again.

Reply:
Hmmm, let's look at this together sometime today and chat.

Reply:
My understanding is these names should not be updated, as they are column names in the DB